tests/NSubstitute.Benchmarks/NSubstitute.Benchmarks.csproj

```diff
@@ -6,7 +6,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="BenchmarkDotNet" Version="0.13.10" />
+    <PackageReference Include="BenchmarkDotNet" Version="0.13.12" />
```

Comment:
looks unrelated to NUnit.

Reply:
yes, this related to Update nuget packages for test projects